From decf01b5740cc6b74c33c38284b80dd13bc20f96 Mon Sep 17 00:00:00 2001
From: jordankooyman <jordankooyman@gmail.com>
Date: Mon, 23 Feb 2026 11:10:38 -0700
Subject: [PATCH] Update LDD Build to only do misc-modules and scull

---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7b08c49..c4f2e2e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,5 @@
 
-SUBDIRS =  misc-progs misc-modules \
-           skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
-	   pci usb\
-	   sbull snull short
+SUBDIRS =  misc-modules scull
 
 all: subdirs
 
